packages/botonic-react/src/app.jsx

```diff
@@ -53,7 +58,8 @@ export class App {
     else return <Webchat botonicApp={this} theme={themeOptions} />
   }
 
-  render(dest, webchatOptions) {
+  render({ dest, webchatOptions, appId }) {
```

Comment:
Dangerous: we're breaking the original signature of this function. I agree it's better to pass an object so we can better handle optional params, but this can break projects with older versions of botonic.

Reply:
You are right, It breaks the older versions because we are expecting specific params. We can make as before, ( render(dest, webchatOptions, appId ) and treat the logic inside the function.

Reply:
Maybe we can pass appId as a webchatOption? In this way we keep the original signature and don't add more params to the function. I also think it make sense from a semantic point of view.

Reply:
I just made a new push with this logic applied!

Reply:
please rebase to master (my branches are on top of this)

Reply:
The branch is up to date with master